README.adoc

```diff
@@ -10,7 +10,7 @@
 :page-layout: guide
 :page-duration: 15 minutes
 :page-releasedate: 2017-09-19
-:page-description: Learn how to build and test a simple web application using Maven and Open Liberty
+:page-description: Learn how to build and test a simple web application using a Maven Project Object Model (POM) file and Liberty Maven plugin.
```

Comment:
"plugin" should be "plug-in."

Add an article ("a" or "the") before "Liberty Maven plug-in."

Reply:
"plugin" is used throughout the entire guide, should I change all those as well?

Reply:
Yes, the IBM style guide says that "plug-in" should be hyphenated. Sorry I did not catch that earlier!

Reply:
No worries, just changed it all.

Reply:
We need to look into this further.  We used plugins here: https://github.com/WASdev/ci.maven

Reply:
Here's a link to the IBM style word usage page where I found the info about "plugin" vs. "plug-in": https://apps.na.collabserv.com/wikis/home?lang=en-us#!/wiki/W580a84a67242_4da6_81cf_69e3db4bdbfa/page/Word%20usage The word "plugin" has its own entry. The Merriam Webster online dictionary also seems to prefer "plug-in."

However, "plugin" is OK when it is used as a name and/or formatted as part of the code, as it is with "liberty-maven-plugin" in the link you mentioned (https://github.com/WASdev/ci.maven). It looks like this ci.maven page uses "plug-in" when the word is not used in reference specifically to "liberty-maven-plugin", which is OK. :)